### PR TITLE
feat: kvradio component was rewritten to use composition api

### PR DESCRIPTION
**Pull request by roger-in-kiva**

### CODE REVIEW

File: @kiva/kv-components/vue/KvRadio.vue [hunk 1]
```diff
@@ -8,14 +8,13 @@
 			<input
 				v-bind="$attrs"
 				:id="uuid"
-				ref="radioRef"
+				:ref="radioRef"
```
Comment (emuvente) [inline @kiva/kv-components/vue/KvRadio.vue:11]: I think this also needs to change back to `ref="radioRef"`

Comment (roger-in-kiva) [reply @kiva/kv-components/vue/KvRadio.vue]: Ready!

**Review by ryan-ludwig — APPROVED** [review]

File: @kiva/kv-components/vue/KvRadio.vue [hunk 2]
```diff
@@ -123,45 +127,54 @@ export default {
...
+		});
+
+		onMounted(() => {
+			uuid = `kvr-${nanoid(10)}`;
+		});
+
+		const onChange = (event) => {
+			emit('change', event.target.value);
+			emit('update:modelValue', event.target.value);
+			emit('update:checked', event.target.value);
```
Comment (ryan-ludwig) [inline @kiva/kv-components/vue/KvRadio.vue:163]: I believe this line can be removed

Comment (emuvente) [reply @kiva/kv-components/vue/KvRadio.vue:163]: +1

Comment (roger-in-kiva) [reply @kiva/kv-components/vue/KvRadio.vue]: Good, it was removed.

**Comment by emuvente** [comment]
@roger-in-kiva please work on getting the tests passing that I added to this PR. The tests can be run with `npm test -w test-vue2 -w test-vue3 KvRadio`, and KvSwitch can be used as an example for getting the tests passing.

**Comment by roger-in-kiva** [comment]
Ready for testing!

**Review by emuvente — APPROVED** [review]
Looks great!

**Pull request merged**